.gitignore

```diff
@@ -9,3 +9,4 @@ gradle/*
 # files
 gradlew
 gradlew.bat
+.env
```

Comment:
According to your git commit messages, you have deleted the env file. Therefore, remove this line from your gitignore as well.

Reply:
I will still need the .env file which contains my Cryptocompare API keys 
So I will leave it in the gitignore file